### ISSUE TITLE
Avatar in other formats aren't loaded

### ISSUE THREAD

**Issue by Greg-Nexus** [Nexus-Mods/NexusMods.App#525]
# Improvement

## Currently 
After logging in to the App via Nexus Mods, we do not display the user's avatar.

## We want
To display the user's avatar when logged-in.

## Supporting Information
![No Avatar](https://github.com/Nexus-Mods/NexusMods.App/assets/100707075/d42799de-e52d-4389-aea7-e43c2736242c)

```
00:00:00.771 [ERROR] Failed to load image: https://forums.nexusmods.com/uploads/profile/photo-thumb-138183793.png|System.Net.Http.HttpRequestException: Response status code does not indicate success: 404 (Not Found).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at System.Net.Http.HttpClient.GetByteArrayAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at NexusMods.App.UI.Controls.TopBar.TopBarViewModel.LoadImage(Uri uri) in /_/src/NexusMods.App.UI/Controls/TopBar/TopBarViewModel.cs:line 51
```

**Comment by erri120** [comment]
![2023-08-07_14-25-57](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/62ab650b-2e52-4abc-8ea6-8946a60010c1)

We already display the user's avatar. This is likely a bug, does the log contain any details?

**Comment by Greg-Nexus** [comment]
Error log added

**Comment by erri120** [comment]
The issue is that our code currently assumes every profile picture uses the PNG format. My profile picture is https://forums.nexusmods.com/uploads/profile/photo-thumb-6672467.png but yours is https://forums.nexusmods.com/uploads/profile/photo-thumb-138183793.jpg.

**Comment by erri120** [comment]
I believe Vortex uses a different API than we do, which returns the profile icon URL. They also have a fallback avatar.

https://github.com/Nexus-Mods/Vortex/blob/b80d5b6d1b53e0dc6e0199c5371394261f8040b9/src/extensions/nexus_integration/views/LoginIcon.tsx#L135-L137

**Comment by erri120** [comment]
https://users.nexusmods.com/oauth/userinfo can be used to get the avatar url:

```json
{
	"sub": "6672467",
	"name": "erri120",
	"email": "",
	"avatar": "https://forums.nexusmods.com/uploads/profile/photo-thumb-6672467.png?r_=1683622107",
	"group_id": 4,
	"membership_roles": [
		"member",
		"supporter",
		"premium",
		"lifetimepremium"
	],
	"premium_expiry": 0
}
```

**Comment by erri120** [comment]
Fixed in #666.
